src/kili/llm/services/export/dynamic.py

```diff
@@ -192,8 +195,8 @@ def _format_comparison_annotation(annotation, completions, job, obfuscated_model
 
 def _format_json_response(
     jobs_config: Dict, annotations: List[Dict], completions: List[Dict], obfuscated_models: Dict
-) -> Dict[str, Union[str, List[str]]]:
-    result = {}
+) -> Dict[str, Dict]:
```

Comment:
Don't you loose some of the typing here ?

Reply:
Yes I will update the type